components/ui/AppPageHeader.vue

```diff
@@ -0,0 +1,188 @@
+      </div>
+
+      <div v-if="hasAside" class="app-page-header__aside">
+        <slot name="aside" />
+      </div>
+    </div>
+  </header>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
+import { GeneralLink } from '~/constants/appLinks'
+
+@Component
+export default class AppPageHeader extends Vue {
+  @Prop({ default: false, required: false, type: Boolean }) bgGrid!: Boolean
+  @Prop({ default: null, required: false, type: Object }) cta!: GeneralLink
+  @Prop({ default: false, required: false, type: Boolean }) fixedHeight!: Boolean
+
+  get hasAside () {
+    return !!this.$slots.aside
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import "~carbon-components/scss/globals/scss/typography";
+@import "~/assets/scss/blocks/copy.scss";
+
+$top-link-height: 70px;
+$top-link-height-large: 60px;
+
+.app-page-header {
+  &__container {
+    @include contained();
+    display: grid;
+    column-gap: $spacing-07;
+    grid-template-columns: 1fr;
+    grid-template-rows: $top-link-height auto;
+    grid-template-areas:
+      "top-link"
+      "main";
+
+    @include mq($until: large) {
+      grid-template-rows: $top-link-height-large auto;
+    }
+
+    @include mq($until: medium) {
+      grid-template-areas:
+        'top-link'
+        'main';
+    }
+
+    &_has-aside {
+      grid-template-columns: 1fr auto;
+      grid-template-areas:
+        "top-link top-link"
+        "main aside";
+
+      @include mq($until: large) {
+        grid-template-columns: repeat(2, 1fr);
+      }
+
+      @include mq($until: medium) {
+        grid-template-columns: 1fr;
+        grid-template-areas:
+          'top-link'
+          'main'
+          'aside';
+      }
+    }
+
+    &_fixed-height {
+      align-items: center;
+      height: 28rem;
+
+      @include mq($until: large) {
+        height: 28rem * 40 / 64;
+      }
+    }
+  }
+
+  &__top-link {
+    align-items: center;
+    display: flex;
+    grid-area: top-link;
+  }
+
+  &__main {
+    display: flex;
+    flex-flow: column;
+    gap: $spacing-05;
+    grid-area: main;
+    justify-content: space-between;
+
+    @include mq($until: x-large) {
+      gap: $spacing-06;
+    }
+
+    @include mq($until: large) {
+      gap: $spacing-09;
+    }
+
+    &_fixed-height {
+      margin-bottom: $top-link-height;
+
+      @include mq($until: large) {
+        margin-bottom: $top-link-height-large;
+      }
+
+      @include mq($until: medium) {
+        margin-bottom: initial;
+      }
+    }
+  }
+
+  &__headline {
+    @include type-style('expressive-heading-05', true);
+    color: $white-text-01;
+    max-width: 10 * $column-size-large;
+
+    @include mq($from: medium, $until: large) {
+      max-width: 6 * $column-size-medium;
+    }
+
+    @include mq($until: medium) {
+      max-width: 5 * $column-size-medium;
+    }
+  }
+
+  &__description {
+    @include type-style("body-long-01");
```

Comment:
You can remove this line, you have already this typography when using `copy__paragraph`